lib/param_parsing/toml.go

```diff
@@ -34,11 +35,14 @@ func getParamsTOML(params Params) (Params, error) {
 		if viperParser.Get("version") != nil {
 			params.Version = viperParser.GetString("version")
 		}
+		if configKey := "product"; viperParser.Get(configKey) != nil {
```

Comment:
Maybe add a note to https://tfswitch.warrensbox.com/usage/config-files/#use-tfswitchtoml-file-for-non-admin-users-with-limited-privilege-on-their-computers ?

Reply:
Fixed in e8e0707